browser-extension/api-server/package.json

```diff
@@ -24,6 +24,7 @@
     "mysql2": "^2.3.3",
     "password-generator": "^2.3.2",
     "sequelize": "^6.14.1",
+    "sequelize-cli": "^6.6.1",
```

Comment:
so either we add it to dockerfile or to package.json. no need to keep it at both places. since when you run npm install all packages mentioned in package.json will get installed, we don't have to then run `npm install sequelize-cli`

Reply:
Keeping install here in package file. Removed from entrypoint script.